accelerometer/summariseEpoch.py

```diff
@@ -180,71 +176,6 @@ def get_interrupts(e, epochPeriod, summary):
-        timeZone='Europe/London'):
-    """Check if data occurs at a daylight savings crossover
-
-    If daylight savings crossover, update times after time-change by +/- 1hr.
-    Also, if Autumn crossover time, remove last 1hr chunk before time-change.
-
-    :param pandas.DataFrame e: Pandas dataframe of epoch data
-    :param datetime startTime: Remove data before this time in analysis
-    :param datetime endTime: Remove data after this time in analysis
-    :param dict summary: Output dictionary containing all summary metrics
-    :param str timeZone: timezone in country/city format to be used for daylight 
-        savings crossover check
-
-    :return: Write dict <summary> key 'quality-daylightSavingsCrossover'
-    :rtype: void
-
-    :return: Update DataFrame <e> time column after time-change crossover.
-    :rtype: void
-    """
-
-    daylightSavingsCrossover = 0
-    utc_tz = pytz.timezone("UTC")
-    localTime = pytz.timezone(timeZone)
-    # Convert because pytz can error if not using python datetime type
-    startTimeZone = localTime.localize(startTime.to_pydatetime())
-    endTimeZone = localTime.localize(endTime.to_pydatetime())
-    if startTimeZone.dst() != endTimeZone.dst():
-        daylightSavingsCrossover = 1
-        # Find whether clock needs to go forward or back
-        if endTimeZone.dst() > startTimeZone.dst():
-            offset = 1
-        else:
-            offset = -1
-        print('different timezones, offset = ', str(offset))
-        # Find actual crossover time
-        for t in localTime._utc_transition_times:
-            if t>startTime:
-                transition = t
-                transition = transition - pd.DateOffset(seconds = 1)
-                transition = utc_tz.localize(transition)
-                transition = localTime.normalize(transition)
-                transition = transition.replace(tzinfo = None)
-                transition = transition + pd.DateOffset(seconds = 1)
-                break
-        # If Autumn crossover time, adjust transition time plus remove 1hr chunk
-        if offset == -1:
-            # Remove last hr before DST cut, which will be subsequently overwritten
-            e = e[(e.index < transition - pd.DateOffset(hours=1)) |
-                    (e.index >= transition)]
-        print('day light savings transition at:', str(transition))
-        # Now update datetime index to 'fix' values after DST crossover
-        e['newTime'] = e.index
-        e['newTime'] = np.where(e.index >= transition,
-                e.index + np.timedelta64(offset,'h'), e.index)
-        e['newTime'] = np.where(e['newTime'].isnull(), e.index, e['newTime'])
-        e = e.set_index('newTime')
-        # Reset startTime and endTime variables
-        startTime = pd.to_datetime(e.index.values[0])
-        endTime = pd.to_datetime(e.index.values[-1])
-    # And record to output summary
-    summary['quality-daylightSavingsCrossover'] = daylightSavingsCrossover
-    return e
-
-
```

Comment:
We no longer need all this

Reply:
The only part I'd like to keep is the 'quality-daylightSavingsCrossover' flag, if possible.

Reply:
Oh and my reason for this is that it would be helpful in subsequent health association analysis for sleep and circadian rhythm use cases i.e. we may like to remove such people from our analysis (assuming their sleep/circadian status is abnormally changed during a daylight saving crossover week)

Reply:
Ahhh right, that should be maintained. I will submit a fix soon.